models/Staticroute.php

```diff
@@ -77,6 +77,11 @@ class Staticroute extends AbstractModel
      */
     public $siteId;
 
+    /**
+     * @var array|string
```

Comment:
can this be a string?

Reply:
yes, cause it is stored as a string, the getter converts it to an array

Reply:
actually, not. you are right. I'll change it

Reply:
changed it